Add procedure for republishing repo (#2252)
theforeman/foreman-documentation: guides/common/modules/proc_republishing-repository-and-content-view-metadata.adoc

[id="Republishing_Repository_and_Content_View-Metadata_{context}"]

> = Republishing Repository and Content View Metadata

Reviewer: I am unsure about two procedures in one file. What do you think? @bangelic @adamlazik1

Author: Should we make republishing Content View metadata its own procedure? It'll be a very quick one but I'm ok with it if you think it makes more sense to have it as a separate procedure.

Reviewer 2: > I am unsure about two procedures in one file. What do you think? @bangelic @adamlazik1

Yeah, that the other thing that I wanted to point out. Probably better to have that in a separate file.

> In the case of republishing repository metadata, this is used when a repository distribution does not have the content it should be distributing based on the contents of the repository.

Reviewer 3: ```suggestion
In the case of republishing repository metadata, this is used when a repository distribution does not have the content that should be distributed based on the contents of the repository.
```

> In the case of republishing repository metadata, this is used when a repository distribution does not have the content that should be distributed based on the contents of the repository.

Reviewer 2: ```suggestion
Republishing repository metadata  is used when a repository distribution does not have the content that should be distributed based on the contents of the repository.
```

> This procedure should be used with caution.

Reviewer 2: ```suggestion
Use this procedure with caution.
```

> In most cases, a complete repository sync or publishing a new Content View version is recommended.

Reviewer 2: ```suggestion
In most cases, {Team} recommends a complete repository sync or publishing a new Content View version.
```

> This procedure should be used with caution, and in most cases, a complete repository sync or publishing a new Content View version are the best options.

Reviewer 3: ```suggestion
This procedure should be used with caution.
In most cases, a complete repository sync or publishing a new Content View version is recommended.
```

.Procedure

. In the {ProjectWebUI}, navigate to *Content* > *Products*.
. Select the product that includes the repository for which you want to republish metadata.

> Navigate to the repository where you want to republish metadata.

Reviewer 2: ```suggestion
. Navigate to the repository where you want to republish metadata.
```

> Navigate to the repository where you wish to republish metadata.

Reviewer 3: ```suggestion
Navigate to the repository where you want to republish metadata.
```

+
[NOTE]
====

> This action is not available for repositories that implement the *Complete Mirroring* policy since the metadata is copied from the repository's upstream source rather than being generated locally.

Reviewer 3: ```suggestion
This action is not available for repositories that implement the *Complete Mirroring* policy since the metadata is copied from the repository's upstream source instead of being generated locally.
```

This action is not available for repositories that implement the *Complete Mirroring* policy since the metadata is copied from the repository's upstream source instead of being generated locally.
====

To republish Content View metadata:

. In the {ProjectWebUI}, navigate to *Content View* > *Version* > *Actions*.

> . Republishing repository metadata will regenerate metadata for all repositories in the Content View version that do not adhere to the *Complete Mirroring* policy.

Reviewer 2: Isn't this missing a step in between?

Author: @sjha4 Is there a step missing in between here?

Reviewer 4: So you go to CV > Version > Details > Actions kebab and the action is there.
Screenshot:
![Screenshot from 2023-06-26 14-54-42](https://github.com/theforeman/foreman-documentation/assets/21146741/979dbe0b-5217-46aa-8806-1fa91f415485)

Author: Steps updated.
